fxmanifest.lua

```diff
@@ -26,7 +26,9 @@ server_scripts {
 
 files {
     'config/shared.lua',
+    'config/client.lua',
+    'config/server.lua'
```

Comment:
Remove config/server.lua as is not needed.

Reply:
Why would it not be needed? server/main.lua calls it

Reply:
The server always has access to all the files within a resources directory. The files directive only tells the client what they can see and download

Reply:
This was my mistake in my latest commit so my apologies but to answer your question, files are files downloaded by the client upon resource start. server files are not needed in here. server/main.lua will see it upon resource start.